api/v1beta1/keystoneapi_types.go

```diff
@@ -125,6 +125,11 @@ type KeystoneAPISpecCore struct {
 	// TrustFlushSuspend - Suspend the cron job to purge trusts
 	TrustFlushSuspend bool `json:"trustFlushSuspend"`
 
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:default=1
+	// FernetRotationDays - Rotate fernet token keys every X days
+	FernetRotationDays *int32 `json:"fernetRotationDays"`
```

Comment:
I would set a minimum to 1

Reply:
Done, I also set the minimum number of fernet keys to 3